data/reusables/getting-started/create-a-repository.md

```diff
@@ -0,0 +1 @@
+For more information on how to create a repository for your projects, see “[Create a repository](/get-started/quickstart/create-a-repo)."
```

Comment:
It would be nice to rephrase this so that it has a similar format to the rest of the reusable bullets. Something like:

```suggestion
Creating a repository for your project allows you to store code in {% data variables.product.prodname_dotcom %}. This provides a backup of your work that you can choose to share with other developers. For more information, see “[Create a repository](/get-started/quickstart/create-a-repo)."
```